docs/index.rst

```diff
@@ -306,6 +306,9 @@ Disks
     (in milliseconds)
   - **busy_time**: (*Linux*, *FreeBSD*) time spent doing actual I/Os (in
     milliseconds)
+  - **time_in_queue**: (*Linux*) total wait time for all requests (in
+    milliseconds)
+  - **in_flight**: (*Linux*) number of I/Os currently in flight
```

Comment:
In order to be consistent with the established naming convention this field should be called `in_flight_count`.
`in_flight_count` looks like a good name (well, I don't like it, but I can't think of anything better).
Also, change the explanation to something more meaningful such as:
"The number of I/O requests that have been issued to the device driver but have not yet completed."